pkg/rpc/client/rpc_test.go

```diff
@@ -0,0 +1,1502 @@
+package client
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/core"
+	"github.com/nspcc-dev/neo-go/pkg/core/block"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/rpc/response/result"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type rpcClientTestCase struct {
+	name           string
+	invoke         func(c *Client) (interface{}, error)
+	serverResponse string
+	result         func(c *Client) interface{}
+	check          func(t *testing.T, c *Client, result interface{})
+}
+
+var rpcClientTestCases = map[string][]rpcClientTestCase{
+	"getaccountstate": {
+		{
+			name: "positive",
+			invoke: func(c *Client) (interface{}, error) {
+				return c.GetAccountState("")
+			},
+			serverResponse: `{"jsonrpc":"2.0","id": 1,"result":{"version":0,"script_hash":"0x1179716da2e9523d153a35fb3ad10c561b1e5b1a","frozen":false,"votes":[],"balances":[{"asset":"0xc56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b","value":"94"}]}}`,
```

Comment:
It'd be nice to add some comment about where this test data comes from either in the commit message or above `rpcClientTestCases` definition.